src/utils.ts

```diff
@@ -5,6 +5,9 @@ import { Readable } from "stream";
 import { configstore } from "./configstore";
 import { FirebaseError } from "./error";
 import * as logger from "./logger";
+import * as winston from "winston";
+import { SPLAT } from "triple-beam";
+const ansiStrip = require("cli-color/strip") as (input: string) => string;
```

Comment:
Since these are all external modules, I'd group them above with the other external modules.

Reply:
good catch, auto import always doing me dirty